doc/neps/nep-0008-groupby_additions.rst

```diff
@@ -1,8 +1,8 @@
 .. _NEP08:
 
-=============================================================
-NEP 8 —  A proposal for adding groupby functionality to NumPy
-=============================================================
+============================================================
+NEP 8 — a proposal for adding groupby functionality to NumPy
```

Comment:
```suggestion
NEP 8 — A proposal for adding groupby functionality to NumPy
```
You didn't change the others, and this is just as well.

Reply:
Thanks. I missed this when going back and fixing false positives.